### PR TITLE
Remove html formbuilder from backend

### PR DESCRIPTION
**Pull request by guerler**
This PR strips the backend form building modules from Galaxy. These features are not used anymore by Galaxy but are embedded in the Toolshed which in some occasions re-uses makos within Galaxy. The solution suggested in this PR is to move form builders for select and checkbox fields to Toolshed makos and use the simple, non-html producing form objects of Galaxy. It also resolves the `True=False` issue related to Toolshed checkboxes. Now `True` equals `True` and `False` equals `False`. This PR is a prerequisite for the removal of the grid HTML builders. Once those are also moved to the Toolshed, Galaxy's backend will not create any HTML anymore.

### CODE REVIEW

**Comment by guerler** [comment]
@galaxybot test this

**Comment by guerler** [comment]
Thanks for the quick fix @davebx.

File: lib/tool_shed/grids/repository_grids.py [hunk 1]
```diff
@@ -1331,7 +1335,11 @@ def get_value(self, trans, grid, repository):
             """Display a SelectField whose options are the changeset_revision strings of all download-able revisions of this repository."""
             select_field = grids_util.build_changeset_revision_select_field(trans, repository, downloadable=True)
             if len(select_field.options) > 1:
-                return select_field.get_html()
+                tmpl = "<select name='%s'>" % select_field.name
+                for o in select_field.options:
+                    tmpl += "<option value='%s'>%s</option>" % (o[1], o[0])
```
Comment (jmchilton) [inline lib/tool_shed/grids/repository_grids.py:1340]: Escape option value at least?

Comment (guerler) [reply lib/tool_shed/grids/repository_grids.py:1340]: Thanks @jmchilton.

**Comment by guerler** [comment]
Thanks a lot for you help @davebx

**Pull request merged**